### PR TITLE
release: create 3.5.0 release notes (PROJQUAY-1489)

### PR DESCRIPTION
**Pull request by thomasmckay**

### CODE REVIEW

File: modules/rn_3_50.adoc [hunk 1]
```diff
@@ -0,0 +1,44 @@
+[[rn-3-500]]
+== Version 3.5.0
+
+=== quay / clair / quay-builder
+
+Note: Some features of Quay are not currently available when running on a FIPS-enabled OCP cluster or RHEL system:
+* FEATURE_MAILING for user create validation and vulnerability notifications will not work
```
Comment (thomasmckay) [inline modules/rn_3_50.adoc:7]: @jonathankingfc Any other config.yaml options impacted?

**Comment by thomasmckay** [comment]
@gabriel-rh this is ready to go

**Comment by LiZhang19817** [comment]
Pls also note that export logs via email also not work.
https://issues.redhat.com/browse/PROJQUAY-1816

**Comment by gabriel-rh** [comment]
Just wondering if the individual notes should reference the corresponding JIRAs?

**Comment by thomasmckay** [comment]
> Just wondering if the individual notes should reference the corresponding JIRAs?
> 

For 3.5 we'll leave out the jira links. With out new changelog automation we can consider it for 3.6.

I'll update w/ above comments

**Review by gabriel-rh — APPROVED** [review]

**Pull request merged**